src/scripts/actions/VideoActions.js

```diff
@@ -3,22 +3,38 @@
 import { createAction } from 'redux-actions'
 
 import paratii from 'utils/ParatiiLib'
-import { VIDEO_SELECT, INIT_VIDEOSTORE } from 'constants/ActionConstants'
+import {
+  VIDEO_SELECT,
+  INIT_VIDEOSTORE,
+  VIDEOFECTH_ERROR,
+  VIDEOFECTH_SUCCESS
```

Comment:
very small deal, but `VIDEOFECTH` is misspelled in both actions 😁 

`VIDEOFECTH => VIDEOFETCH`

Reply:
i've change it in the new commit!